lib/iris/tests/results/netcdf/netcdf_monotonic.cml

```diff
@@ -48,7 +48,7 @@
         <dimCoord id="62e940e0" points="[10, 20, 30]" shape="(3,)" standard_name="longitude" units="Unit('degrees')" value_type="int32" var_name="lon"/>
       </coord>
       <coord datadims="[0]">
-        <auxCoord id="cb784457" points="[  1, 100,   3]" shape="(3,)" standard_name="time" units="Unit('hours since 1970-01-01 00:00:00', calendar='standard')" value_type="int32" var_name="time3"/>
+        <auxCoord id="cb784457" points="[  1, --,   3]" shape="(3,)" standard_name="time" units="Unit('hours since 1970-01-01 00:00:00', calendar='standard')" value_type="int32" var_name="time3"/>
```

Comment:
This may look like I'm just changing the test to suit me! But I believe my changes have actually improved how Iris works. Our previous graceful fallback on to `AuxCoord` involved using the `DimCoord` arguments for `AuxCoord` creation - so no masking - whereas we are now fully retrying the whole process if `DimCoord` fails.